src/lib/dom.ts

```diff
@@ -24,6 +24,7 @@ interface MaybePolyfilledCe extends CustomElementRegistry {
  * True if the custom elements polyfill is in use.
  */
 export const isCEPolyfill = window.customElements !== undefined &&
+    window.customElements !== null &&
```

Comment:
We can change this to a single check:

```js
export const isCEPolyfill = window.customElements != null &&
```

This'll check for both `null` and `undefined`, since they are loosely equal to each other.

Reply:
True, I can make that change.